catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -213,7 +195,6 @@ const SearchLayout: FunctionComponent<{ hasEventsExhibitions: boolean }> = ({
             });
 
             updateUrl(event.currentTarget);
-            return false;
```

Comment:
https://reactjs.org/docs/handling-events.html

I think this was not necessary based on:

Another difference is that you cannot return false to prevent default behavior in React. You must call preventDefault explicitly. For example, with plain HTML, to prevent the default form behavior of submitting, you can write:
```
<form onsubmit="console.log('You clicked submit.'); return false">
  <button type="submit">Submit</button>
</form>
```

In React, this could instead be:

```
function Form() {
  function handleSubmit(e) {
    e.preventDefault();
    console.log('You clicked submit.');
  }

  return (
    <form onSubmit={handleSubmit}>
      <button type="submit">Submit</button>
    </form>
  );
}
```